app/controllers/image_uploads_controller.rb

```diff
@@ -46,6 +53,12 @@ def cloudinary_link(uploaders)
 
   private
 
+  def valid_filename?
+    images = Array.wrap(params.dig("image"))
+    invalid_image = images.detect { |image| long_filename?(image) }
```

Comment:
Ruby has `Enumerable#none?`, which I think reads quite nicely here (you then won't need the last line, as the result of `none?` is the boolean we want):

```suggestion
    images.none? { |image| long_filename?(image) }
```

Reply:
Love it - updated! Always game to be reminded of the nifty Ruby methods 💯. Thank you!